CHANGELOG.md

```diff
@@ -8,7 +8,13 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
-## [v0.9.0-beta] - 2019-04-16
+## [v1.0.0+dogwood] - 2021-02-11
```

Comment:
Is this intended to have a `+` sign here ?

Reply:
Yes, this is part of pypi-allowed release name patterns

Reply:
Ok, thanks!

Reply:
Reference: https://www.python.org/dev/peps/pep-0440/#local-version-identifiers